src/scripts/main.js

```diff
@@ -1,3 +1,92 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+const rowAppend = document.querySelector('.append-row');
+const rowRemove = document.querySelector('.remove-row');
+const columnAppend = document.querySelector('.append-column');
+const columnRemove = document.querySelector('.remove-column');
+
+const actions = {
+  'append-row': appendRow,
+  'remove-row': removeRow,
+  'append-column': appendColumn,
+  'remove-column': removeColumn,
+};
+
+const tr = tbody.querySelectorAll('tr');
```

Comment:
The `tr` variable is initialized once and does not update when rows are added or removed. Consider moving this query inside the `appendColumn` and `removeColumn` functions to ensure it always reflects the current state of the table.